gulpfile.babel.js

```diff
@@ -1,18 +1,40 @@
 import gulp from 'gulp';
 import requireDir from 'require-dir';
-import livereload from 'gulp-livereload';
+import browserSync from 'browser-sync';
 
 requireDir( './gulp-tasks' );
 
+const packageJson = require('../../../package.json');
+const localURL = packageJson.testing.urls.local;
+const bs = browserSync.create();
```

Comment:
@colorful-tones since this is dependent on the the local URL being present we should probably run a check on the object before executing the tasks

Reply:
@timwright12 I revisited how the BrowserSync instance is created and made it conditional. I mostly gleaned from [WP Rig](https://github.com/wprig/wprig) as they had a unique solution for this issue.